### ISSUE TITLE
Why does the Arabic version of the arabic_arabian_peninsula sky culture have such a different description?

### ISSUE THREAD

**Issue by 10110111** [Stellarium/stellarium#3712]
First, what made me wonder is why the section structure is completely different in [ar](https://github.com/Stellarium/stellarium/blob/089928d0cfeb39333b1f76709427b4aa7fd02d4c/skycultures/arabic_arabian_peninsula/description.ar.utf8) vs [en](https://github.com/Stellarium/stellarium/blob/089928d0cfeb39333b1f76709427b4aa7fd02d4c/skycultures/arabic_arabian_peninsula/description.en.utf8) versions. Apparently, one of the reasons is that the Arabic text doesn't need the transliterations and translations present for the other langugages, which might justify the switch from a table to a bunch of `<h4>` sections.

But, the text itself also differs considerably. Compare, e.g. the text for constellation "النعايم":

English:

> | ID | Arabic | Transliteration | Translation | Comment |
> |----|--------|-----------------|-------------|---------|
> | CON 200 | النعايم | al-Naʿāyem | Al-Nayem | Meaning is not known for certain. It could mean the camels. composed of the seven stars of the Big Dipper. |

Arabic:

> **النعايم - السبَّع - بنات نعش**
> سبعة نجوم شهيرة شمالية. ويرد اسم النعايم كثيرا في الشعر العامي

Translated from Arabic (by Google Translate):

> **Al-Na’aym - Al-Saba’ - Daughters of a Coffin**
> Seven famous northern stars. The name Al-Na'ayim appears frequently in colloquial poetry

Same mismatch exists for some (many?) others.

I've also seen some mismatching descriptions of the Chinese culture in translations like French, which are much shorter (and have fewer sections) than the English one.

I'm currently in the process of converting the current sky cultures to the format used in [stellarium-skycultures](https://github.com/Stellarium/stellarium-skycultures) repo (which would make it possible to avoid too much change at once when switching to this format), and this situation puzzles me regarding how I should proceed. Should I maybe just drop all these old cultures and simply import the ones from that repo, converting only the missing ones?

**Comment by alex-w** [comment]
Sorry for late answer.

The situation is existing by historical reasons. Previously all translations has been made by hands and not all translations were synced. After switching to Transifex the situation coming better, but again - not all translations for sky cultures were updated/added via Transifex.

I fear the question of priority of description for each sky culture should be reviewed separately, because new format are missing some features/data, were coming in desktop planetarium later introducing of new format of SC. Plus, the names of regions and licenses are different too.

It is possible that adding support new format of SC in desktop planetarium will cause some changes in all planetariums :(

@xalioth @gzotti @guillaumechereau @sushoff

**Comment by 10110111** [comment]
> new format are missing some features/data

What is missing? So far I haven't found any serious missing part (except minor things like classification).

> Plus, the names of regions and licenses are different too.

Examples?.. What's the difference in licensing except the _"Special pemission is granted to Stellarium Labs to include the licensed material."_ present in e.g. Arabic cultures and lacking in their desktop counterparts?

**Comment by 10110111** [comment]
Some of the issues I've solved in my [`skycultures-new-format`](https://github.com/Stellarium/stellarium/tree/skycultures-new-format) branch. Particularly, the licenses are also parsed and displayed there, just as classification (which I added in [my fork of SSC](https://github.com/10110111/stellarium-skycultures)).

The differences you mention like the region are one of the reasons I was planning to first convert the existing cultures to the new format, and only later to replace them with the ones from SSC (maybe having altered them to conform to the requirements). This is why I came across this issue with the translations.

**Comment by sushoff** [comment]
ugh! 

well, translations are always a snapshot in time! If somebody is kind enough to translate a description and then, later, the original author changes the description in the original language (but of course not in all languages), the translation(s) start to deviate. It will always happen because research goes on, we (researchers) update as many languages as we can (at least I do) but when I don't understand a language, I don't dare to change it. ... Our problem of software developers/administrators will always be that we never know that the original language of the researcher is (e.g.: I may possibly write in English / French directly or - on bad days when the language regionn of my brain doesn't work properly - in German ... so you as a developer cannot even guess)   

Of course, as Alex said, this issue is for historical reasons. 

What I currently see for many websites that I administrate: people/ institutions start to implement DEEPL (or: GoogleTranslate), so the author can write only one original (e.g. English) and the translation (to Spanish or whatever) will be done by the machine and on demand.

**Comment by gzotti** [comment]
Is this still really open or did it dissolve with the new SC format?

**Comment by 10110111** [comment]
Well, now this SC simply doesn't have an Arabic translation, so I guess this means the issue itself is dissolved...

**Comment by gzotti** [comment]
I just hope nothing of importance was lost in translation. Presumably the Arab text may have been more complete? @kajaji ? Leaving it open for now, but any missing bits might be restored hopefully by 26.3 by synchronizing (manually:-( in Transifex. I know, large texts on Transifex are no fun...

**Comment by gzotti** [comment]
@kajaji has thankfully sent several updates for 26.2. Is this issue still relevant and to be solved over the summer? Or are we all happy with the current texts and formats? (Unfinished translations on Transifex should not prevent us from closing this.)

**Comment by kajaji** [comment]
I don't think this applies to current implementation. Now we have only description.md. The issue was when there was an option to have description.xx.utf8 for each language.

I'll review the current Transifex translations when time permits.

**Comment by alex-w** [comment]
What is status of this issue?

**Comment by 10110111** [comment]
I think it can be closed. It's now obsolete.
